# an empty map_name will use the currently loaded map
string map_name
---
graph_map/Zone[] zones
bool success
string message